### DISCUSSION TITLE
Scene failed to load

### DISCUSSION THREAD

**Discussion by jessicasymons** [mozilla/hubs#3260] [General Questions & Discussion]
**Description**
Tried to open hub several times on Samsung S2 tablet and on iPhone

Each time, received message 'Scene failed to load'

**To Reproduce**
1. Go to https://hubs.mozilla.com/m6hDQPe/ using phone or tablet
2. . See error

**Expected behavior**
Hub loads

**Screenshots**

**Hardware** 
 - Device: Samsung S2 / iPhone
 - OS: Android / iOS
 - Browser Samsung Internet / Safari

**Additional context**
This hub has some v large assets in it but it does work on some devices e.g. Samsung Galaxy S9
![Screenshot_20201031-224134](https://user-images.githubusercontent.com/54954604/97791536-43847f80-1bcb-11eb-8b39-030c18e9bc6b.png)

**Comment by misslivirose** [comment]
This didn't reproduce for me on an iPhone 11 Pro, but I suspect that this is because the scene is much larger than we recommend for accessing on phones/tablets. There are 40 textures and 713,188 triangles, both of which can contribute to performance problems. The recommended polygon count for mobile devices is 50,000 triangles. It looks like this floor on this room was created by pasting a lot of images into the Hubs room - there might be a performance improvement that can be made by using Spoke to place a textured floor, rather than having them all be added as separate images, and the publishing workflow in Spoke will do a performance check on the scene. 

I'm going to convert this to a discussion so that folks can chime in with additional suggestions for optimization for this particular room/scene.

**Comment by awhite001** [comment]
I encountered the same issue with our school-issued iPads. Not a single one could enter the room. It worked on student's iPhones, but none of the iPads. I'm wondering if this is a setting in Safari that needs to be changed?

**Comment by philtweir** [comment]
I have been having this on a Hubs cloud instance for several months - I can only get in via private browsing, despite clearing my cache and all site data (Chrome 87/88 on Ubuntu). No obvious errors in the network logs (404 from app-icon.png is all) but a number of console log errors. Log (red.) attached.
[hubs.flaxandteal.com-1611915199390.log](https://github.com/mozilla/hubs/files/5893154/hubs.flaxandteal.com-1611915199390.log)

**Reply by philtweir** [reply]
It is consistent so can reproduce on demand.

**Comment by pattersonbl2** [comment]
Are you still having issues with this ?  I was able to access the page using a iPhone 12 Pro max .

**Comment by jessicasymons** [comment]
HI Brandon

I haven't had an issue with this for a while!

---
Thanks
Jess

---------------
Dr Jessica Symons
Director
Visioning Lab Ltd

Email: ***@***.***
Twitter: @jessicasymons
Mobile: 07984747796

Websites: www.visioninglab.com / www.popupview.com

**Comment by nupur711** [comment]
I am facing same issue. Anyone know the resolution

**Comment by pattersonbl2** [comment]
I am not sure if the developer console can be open on mobile but that would be where I would look first.  I am able to load the scene with only one object missing.
